chucks-pick3-mobile/src/app/models/pick3-draw-date-card.domain.spec.ts

```diff
@@ -0,0 +1,155 @@
+import {Pick3DrawDateCardDomain} from "./pick3-draw-date-card.domain";
+import {TestBed} from "@angular/core/testing";
+import {NumberUtilityService} from "../services/numberUtility.service";
+import {Pick3DrawTimeEnum} from "./pick3-draw-time.enum";
+import {Pick3LotteryService} from "../services/pick3-lottery.service";
+import {IonicToastNotificationService} from "../services/ionic-toast-notification.service";
+describe('Pick3DrawDateCardDomain', () => {
+
+  beforeEach(() => {
+
+  });
+  let model = new Pick3DrawDateCardDomain({
+    drawDate : new Date(),
```

Comment:
Create a const for date before using the date here.  And then later you can expect for the exact date.

Reply:
added date and replaced expected